src/js/events/types.ts

```diff
@@ -0,0 +1,40 @@
+// These types were written with reference to the query in fetch-events/index.js and
+// the graphql schema documented at https://www.meetup.com/api/schema
+
+export type EventItem = {
+  readonly event: Event;
```

Comment:
nit: can we apply lexicographical sorting on attributes of types; simply a consistency, should not be considered a requirement for approval however

Reply:
Resolved in [553ddb6](https://github.com/newwwie/newwwie.com/pull/117/commits/553ddb6c839d5619d71c513eba742e433405e367)